internal/sdkprovider/service/kafkatopic/kafka_topic.go

```diff
@@ -289,26 +290,23 @@ func resourceKafkaTopicCreate(ctx context.Context, d *schema.ResourceData, m int
 		Tags:        getTags(d),
 	}
 
-	w := &kafkaTopicCreateWaiter{
-		Client:        m.(*aiven.Client),
-		Project:       project,
-		ServiceName:   serviceName,
-		CreateRequest: createRequest,
+	err = m.(*aiven.Client).KafkaTopics.Create(
+		project,
+		serviceName,
+		createRequest,
+	)
+	if err != nil && !aiven.IsAlreadyExists(err) {
+		diag.FromErr(err)
 	}
 
-	timeout := d.Timeout(schema.TimeoutCreate)
-
-	// nolint:staticcheck // TODO: Migrate to helper/retry package to avoid deprecated WaitForStateContext.
-	_, err = w.Conf(timeout).WaitForStateContext(ctx)
-	if err != nil {
-		return diag.FromErr(err)
+	// Invalidates cache for the topic
+	// We need this only for acceptance tests
+	if os.Getenv("TF_ACC") == "1" {
+		DeleteTopicFromCache(project, serviceName, topicName)
 	}
```

Comment:
This code must be a rudimentary. [This](https://github.com/aiven/terraform-provider-aiven/blob/7b54651089fce0a9cc45db4a36c68291f521a407/internal/sdkprovider/service/kafkatopic/kafka_topic_test.go#L392-L392) should be enough.

Reply:
ok, I'm going to try this one